x-pack/plugin/ilm/qa/multi-node/src/javaRestTest/java/org/elasticsearch/xpack/ilm/TimeSeriesLifecycleActionsIT.java

```diff
@@ -223,6 +228,40 @@ public void testRolloverActionWithIndexingComplete() throws Exception {
             getOnlyIndexSettings(client(), originalIndex).get(LifecycleSettings.LIFECYCLE_INDEXING_COMPLETE)));
     }
 
+    public void testRolloverActionWithMaxSinglePrimarySize() throws Exception {
```

Comment:
`TimeSeriesLifecycleActionsIT` has become quite big and we're moving away from adding to it, towards having one IT/action ( https://github.com/elastic/elasticsearch/issues/53488 )
We can leave this test here for now, but would be awesome to separate the `rollover` tests into a `RolloverActionIT` (in a future PR maybe)

Reply:
+1, but will do as a separate PR

Reply:
https://github.com/elastic/elasticsearch/pull/69295